force-app/main/default/classes/RD2_ERecurringDonationsService.cls

```diff
@@ -38,6 +38,9 @@ public with sharing class RD2_ERecurringDonationsService {
     private final String MONTHLY = 'Monthly';
     private final String WEEKLY = 'Weekly';
     private final String YEARLY = 'Yearly';
+    private final String STATUS_PAUSED = 'Paused';
```

Comment:
These are included in `RD2_Constants` (`STATUS_PAUSED`, etc.)

Reply:
OK Luke!, I'll add it. Thanks.